dev_guide/integrating_external_services.adoc

```diff
@@ -53,12 +53,20 @@ ignore the service and allow you to specify endpoints manually:
 ----
 {
   "kind": "Service",
-  "apiVersion": "v1beta3",
+  "apiVersion": "v1",
   "metadata": {
     "name": "example-mysql-service"
   },
-  "spec": { <1>
```

Comment:
I think there should be added:
```
        "selector": {
        }
```
to which the footnote should be referring to, but I'm not sure if it makes sense. @mfojtik your knowledge is needed in this topic, would adding empty selector match what the footnote is saying?